ui/page/dexclient/create_wallet_widget.go

```diff
@@ -0,0 +1,177 @@
+package dexclient
+
+import (
+	"fmt"
+	"strconv"
+
+	"decred.org/dcrdex/client/asset/btc"
+	"decred.org/dcrdex/client/asset/dcr"
+	"gioui.org/layout"
+	"gioui.org/widget"
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type dexCreateWalletWidget struct {
+	*load.Load
+	sourceAccountSelector *components.AccountSelector
+	backButton            decredmaterial.IconButton
+	createNewWallet       decredmaterial.Button
+	walletPassword        decredmaterial.Editor
+	appPassword           decredmaterial.Editor
+	walletInfoWdg         *walletInfoWidget
+	isSending             bool
+	walletCreated         func()
+}
+
+type walletInfoWidget struct {
+	image    *decredmaterial.Image
+	coinName string
+	coinID   uint32
+}
+
+func newDexCreateWalletWidget(l *load.Load, walInfoWdg *walletInfoWidget, walletCreated func()) *dexCreateWalletWidget {
+	dcw := &dexCreateWalletWidget{
+		Load:            l,
+		walletPassword:  l.Theme.EditorPassword(new(widget.Editor), "Wallet Password"),
+		appPassword:     l.Theme.EditorPassword(new(widget.Editor), "App Password"),
+		createNewWallet: l.Theme.Button("Add"),
+		walletInfoWdg:   walInfoWdg,
+		walletCreated:   walletCreated,
+	}
+
+	dcw.createNewWallet.TextSize = values.TextSize12
```

Comment:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/27733432/144830714-d7b7fe9a-4f93-4161-b26b-fff3dfe4fa83.png">

This widget should be a modal widget see the way it was implemented in staking page https://github.com/planetdecred/godcr/blob/master/ui/page/staking/purchase_modal.go

Reply:
I also dont see a cancel button or a way to go back.